src/main/java/uk/gov/pay/directdebit/payments/model/GoCardlessEvent.java

```diff
@@ -0,0 +1,106 @@
+package uk.gov.pay.directdebit.payments.model;
+
+import java.time.ZonedDateTime;
+
+public class GoCardlessEvent {
+
+    private Long id;
+    private Long paymentRequestEventId;
+    private String eventId;
+    private String action;
```

Comment:
because actions are specific to a resource type, and this probably will call for a slightly more complex design than just sticking an enum there. I figured we would tackle this when actually playing the stories to handle more actions, otherwise this story will be quite more complex than expected

Reply:
Fair enough. As long as we don’t intend to keep `action`s stringly-typed forever!